raster-foundry/raster-foundry
# Don't link external s3 resources

## Overview
We should not create pre-signed download urls for s3 resources in other people's buckets. Link to documentation for the AWS client instead.

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Symlinks from new migrations present or corrected for any new migrations~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

## Testing Instructions

 * Create an import using an s3 url
* Verify that it no longer links directly, and instead had help text / link to documentation on how to download from s3 using credentials
Closes #3881
